packages/bridge-ui-v2/src/libs/bridge/checkBalanceToBridge.ts

```diff
@@ -49,9 +52,14 @@ export async function checkBalanceToBridge({
     } catch (err) {
       console.error(err);
 
-      if (`${err}`.includes('transaction exceeds the balance of the account')) {
+      // TODO: rely on error code, or instance, instead of string matching
```

Comment:
General question, do we have a backlog of these todos? If it's something important we should create an issue for it and add the issue # into the todo. Let's talk about it tomorrow and discuss this. :)

Reply:
At the moment these TODOs live in the code because I'm the only one working on this codebase. The plan is actually to create tickets out of them as you mentioned